src/rest/locationRest.js

```diff
@@ -14,6 +14,17 @@ const api = {
         }
       });
   },
+  fetchLoginLocationLink: () => {
```

Comment:
Let's change this to "fetchHeaderLogoLinks", and instead o putting this in the locationRest service, let's create a new REST service called "appframeworkRest.js".   (The "extension" endpoint is part of the OpenMRS "appframework", which I can explain in more detail sometime).

Reply:
Fixed